Src/WitsmlExplorer.Api/Services/LogCurvePriorityService.cs

```diff
@@ -13,70 +10,85 @@ namespace WitsmlExplorer.Api.Services
 {
     public interface ILogCurvePriorityService
     {
-        Task<IList<string>> GetPrioritizedCurves(string wellUid, string wellboreUid);
-        Task<IList<string>> SetPrioritizedCurves(string wellUid, string wellboreUid, IList<string> prioritizedCurves);
+        Task<IList<string>> GetPrioritizedLocalCurves(string wellUid, string wellboreUid);
+        Task<IList<string>> SetPrioritizedLocalCurves(string wellUid, string wellboreUid, IList<string> prioritizedCurves);
+        Task<IList<string>> SetPrioritizedUniversalCurves(List<string> prioritizedCurves);
+        Task<IList<string>> GetPrioritizedUniversalCurves();
     }
 
-    public class LogCurvePriorityService : ILogCurvePriorityService
+    public class LogCurvePriorityService(
+        IDocumentRepository<LogCurvePriority, string>
+            logCurvePriorityRepository)
+        : ILogCurvePriorityService
     {
-        private readonly IDocumentRepository<LogCurvePriority, string> logCurvePriorityRepository;
-
-        public LogCurvePriorityService(IDocumentRepository<LogCurvePriority, string> logCurvePriorityRepository)
-        {
-            this.logCurvePriorityRepository = logCurvePriorityRepository;
-        }
-
-        public async Task<IList<string>> GetPrioritizedCurves(string wellUid, string wellboreUid)
+        private const string Global = "global";
```

Comment:
Maybe use something like this instead
`private const string UniversalDbId = "universal";`

Reply:
fixed